pypeit/utils.py

```diff
@@ -239,8 +305,7 @@ def bspline_profile(xdata, ydata, invvar, profile_basis, upper=5, lower=5,
             # we'll do the fit right here..............
             if error != 0:
                 bf1, laction, uaction = sset.action(xdata)
-                #if((bf1 == -2) or (bf1.size !=nx*nord)):
-                if np.any(bf1 == -2) or (bf1.size !=nx*nord):
+                if(np.any(bf1 == -2) or (bf1.size !=nx*nord)):
```

Comment:
remove the outer parentheses and fix the white space

Reply:
Done.